portfolio_home.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <title>Jake Rooms - Portfolio</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+	<main>
+		<!--Html for the top navigation bar-->
+		<header class='link_bar'>
+			<a class= 'top_link' href='portfolio_aboutme.html'>About me</a>
+		</header>
+		<!--Html for the image and project navigation bar-->
+		<section class='section_home'>
+			<img class='img_home' src='jake_face.jpg' alt=''/>
+			<!--Html for the brojects navigation menu-->
+			<div class='projects'>
+				<h2>Projects</h2>
+				<nav>
+					<a href='portfolio_project1.html'>Project 1</a>
+					<a href='portfolio_project2.html'>Project 2</a>
+					<a href='portfolio_project3.html'>Project 3</a>
+					<a href='portfolio_project4.html'>Project 4</a>
+				</nav>
+			</div>
+		</section>
+		<!--Html for the footer , contact details and social media buttons-->	
+		<footer class='link_bar'>
+			<!--Html for the contact information-->
+			<div class='contact_info'>
+				<h2>Contact me</h2>
+				<label>Phone: 0000000</label>
```

Comment:
Is label the correct tag here as you have no input?

Reply:
changed to an address tag